KmsClient.js

```diff
@@ -57,11 +58,10 @@ export class KmsClient {
     _assert(type, 'type', 'string');
     _assert(invocationSigner, 'invocationSigner', 'object');
 
-    const keySuite = cryptoLd.suites.get(type);
-    const contextUrl = keySuite.CONTEXT_URL;
+    const {CONTEXT_URL: suiteContextUrl} = cryptoLd.suites.get(type);
```

Comment:
@dlongley @mattcollier hmm I wonder if we may run into trouble here for the `AesKeyWrappingKey2019` and Sha256Hmac key types. (since they won't have suite entries in cryptoLd).
We still generate those via a 'generate key' KMS operation, right?

Reply:
@dmitrizagidulin, yes we generate them that way. So we'll need to handle those cases differently here somehow.

Reply:
@dlongley @dmitrizagidulin Should this now be:

```js
    let contextUrl;
    if(type === 'Sha256HmacKey2019' || type === 'AesKeyWrappingKey2019') {
      contextUrl = WEBKMS_CONTEXT_URL;
    } else {
      const {SUITE_CONTEXT} = cryptoLd.suites.get(type);
      contextUrl = [WEBKMS_CONTEXT_URL, SUITE_CONTEXT];
      // OR should it be just
      // contextUrl = SUITE_CONTEXT; 
    }
```

Reply:
@JSAssassin - close, yeah. Except that the two symmetric types /also/ will need their contexts included, based on type. Something like here: https://github.com/digitalbazaar/bedrock-ssm-mongodb/pull/13/files#diff-3e5dad5d511d3177dfb5a8bd768fe66ab91d52c35ffbb0d50171b00868c80709R46  (which is drawing off of this mapping https://github.com/digitalbazaar/bedrock-ssm-mongodb/pull/13/files#diff-602f56bb692d9bf3bdc5fef3d75d5fa33fe841c3d21ca46294bb6fdaf0785d0cR17 )

Reply:
Also, whoops, I just realized I put the wrong context url there. it should be `'https://w3id.org/security/suites/aes-key-wrapping-2019/v1'` (right now it doesn't have the `/suites/` part)

Reply:
K, all fixed.